go.mod

```diff
@@ -60,6 +60,7 @@ require (
 	github.com/prometheus/client_golang v1.20.5
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
+	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
```

Comment:
This pkg is kind of unmaintained. In runc we are moving away to our fork that we moved to the moby org. I think avoiding it will be nice.